mimesis/providers/personal.py

```diff
@@ -65,7 +65,7 @@ def work_experience(self, working_start_age=22):
 
         return max(a - working_start_age, 0)
 
-    def name(self, gender=None):
+    def name(self, gender: str=None) -> str:
```

Comment:
As i say above, i have created special type for Gender. You should you it. It's added in branch type-hinting`.